src/main/java/hypersquare/hypersquare/plot/PlotDatabase.java

```diff
@@ -201,6 +203,56 @@ public static Component getPlotName(int plotID) {
         return null;
     }
 
+    public static String getRawPlotName(int plotID) {
```

Comment:
whats the reason for this?
dont we always want to show the player the styled plot name?